src/openrct2/localisation/StringIds.h

```diff
@@ -2704,6 +2608,8 @@ enum
     STR_SET_PATROL_AREA = 3445,
     STR_CLEAR_PATROL_AREA = 3446,
 
+    // 3447--5119 unused
```

Comment:
I believe the user strings used to be put here (or still are).

Reply:
I believe they used to, but no longer are. @IntelOrca, do you remember?

Reply:
I don't remember, sorry.

Reply:
I have rephrased the comment to mention user strings. We can free up the block once we are certain it can be used.